docs/src/SUMMARY.md

```diff
@@ -5,67 +5,68 @@
-    - [Querying the blockchain](./providers/querying.md)
-  - [Managing wallets](./getting-started/managing-wallets.md)
-    - [Creating a wallet from a private key](./wallets/private-keys.md)
-    - [Creating a wallet from mnemonic phrases](./wallets/mnemonic-wallet.md)
-    - [Wallet Access](./wallets/access.md)
-    - [Encrypting and storing wallets](./wallets/encrypting-and-storing.md)
-    - [Checking balances and coins](./wallets/checking-balances-and-coins.md)
-    - [Setting up test wallets](./wallets/test-wallets.md)
-    - [Signing](./wallets/signing.md)
-    - [Transferring assets](./wallets/transferring-assets.md)
-  - [Generating bindings with `abigen!`](./getting-started/abigen.md)
-    - [The JSON ABI file](abigen/the-json-abi-file.md)
-    - [The `abigen!` macro](abigen/the-abigen-macro.md)
-  - [Deploying contracts](./getting-started/contracts.md)
-    - [Interacting with contracts](./contracts/interacting-with-contracts.md)
-    - [The FuelVM Binary file](./contracts/the-fuelvm-binary-file.md)
-  - [Calling contracts](./getting-started/calling-contracts.md)
-    - [Connecting wallets](./calling-contracts/calls-with-different-wallets.md)
-    - [Transaction parameters](./calling-contracts/tx-params.md)
-    - [Call parameters](./calling-contracts/call-params.md)
-    - [Call response](./calling-contracts/call-response.md)
-    - [Logs](./calling-contracts/logs.md)
-    - [Variable outputs and messages](./calling-contracts/variable-outputs.md)
-    - [Read-only calls](./calling-contracts/read-only.md)
-    - [Calling other contracts](./calling-contracts/other-contracts.md)
-    - [Multiple contract calls](./calling-contracts/multicalls.md)
-    - [Transaction dependency estimation](./calling-contracts/tx-dependency-estimation.md)
-    - [Estimating cost](./calling-contracts/cost-estimation.md)
-  - [Running scripts](./getting-started/running-scripts.md)
-  - [Predicates](./getting-started/predicates.md)
-    - [Predicate data](./predicates/predicate-data.md)
-    - [Signatures example](./predicates/send-spend-predicate.md)
-  - [Types](./getting-started/types.md)
-    - [Bytes32](./types/bytes32.md)
-    - [Address](./types/address.md)
-    - [ContractId](./types/contract-id.md)
-    - [AssetId](./types/asset-id.md)
-    - [Converting native types](./types/conversion.md)
-    - [Bech32](./types/bech32.md)
-    - [Structs and enums](./types/custom_types.md)
-    - [String](./types/string.md)
-    - [Bits256](./types/bits256.md)
-    - [B512](./types/B512.md)
-    - [EvmAddress](./types/evm_address.md)
-    - [Vectors](./types/vectors.md)
-- [API](./getting-started/api.md)
-- [Debugging](./debugging/debugging.md)
-  - [Debugging abigen errors](./debugging/abigen.md)
-  - [Debugging contract call errors](./debugging/contract-calls.md)
-  - [Debugging network errors](./debugging/network.md)
-  - [The Function selector](./debugging/function-selector.md)
+- [Connecting to a Fuel node](./getting-started/connecting.md)
```

Comment:
Awesome! Now that I see it, I can not understand how nobody noticed it before :smile: